### DISCUSSION TITLE
Font problem

### DISCUSSION THREAD

**Discussion by edbeee** [freeplane/freeplane#2048] [Help]
Hi there, I use Freeplane nearly everyday. It's really a great app no doubt. it made my life easier. 
But I'm facing some issues about the font and export.
When I'm typing in bangla language the lines are automatically breaking when I set the content type "For details" to "Standard" or "text". But when I set that in "latex" it doesn't break. So, what is the problem here?

the first node in the image is set to "Standard" .
![image](https://github.com/user-attachments/assets/4b5b5006-06f5-46bf-abee-b3ea873966ba)

the second node in the following image is set to "Latex"
![image](https://github.com/user-attachments/assets/0379a573-221c-4293-a560-3e635121e544)

You can spot the differences. why is it happening even though the texts I wrote are completely same.

**Comment by aaa1386** [comment]
Watch the video below:


https://github.com/user-attachments/assets/8e09642b-908e-4aee-b290-bdcdae66dcc0


I wrote the 6-letter word "دروازه" in the node.
This word is Arabic.
 After I put a code in the CSS environment, it caused the word "دروازه" to be displayed in two lines. 
It is not interesting at all to display part of a word in one line and another part in the second line.

It is even more strange that when I delete the code in css, the written word is displayed in two lines. That is, I have no way to display the word in the correct node.

Even if I set the node content to "latex", the problem is still not solved.

**Reply by aaa1386** [reply]
A character is always moved to the second line.


https://github.com/user-attachments/assets/7b9f5b48-e573-4218-978b-cf53c2ccb6f1

**Reply by aaa1386** [reply]
The problem I raised was related to when I was using CSS code.
But if I don't use the CSS code, there is no problem in most of the fonts, only a few fonts have this problem. Even if I set the content type to "LaTex" the problem still exists.

**Reply by edbeee** [reply]
I didn't use CSS but I'm facing from problem on the "details" of any node. I don't what is causing this problem.

**Reply by aaa1386** [reply]
Yes, I am facing exactly the same problem when using "Details".

**Reply by aaa1386** [reply]
If this issue is reported as a bug, maybe the FP developer will fix it.

**Reply by edbeee** [reply]
https://github.com/freeplane/freeplane/issues/2055#issue-2554233927

I reported it as the bug but there will be no fix in this case. This is not a bug
